ERCS/erc-7786.md

```diff
@@ -74,6 +74,21 @@ interface IERC7786GatewaySource {
         bytes calldata payload,
         bytes[] calldata attributes
     ) external payable returns (bytes32 outboxId);
+
+    function quoteRelay(
+        string calldata destinationChain,
+        string calldata receiver,
+        bytes calldata payload,
+        bytes[] calldata attributes
+        uint256 gasLimit,
+        string[] calldata refundReceivers
+    ) external returns (uint256);
+
+    function requestRelay(
```

Comment:
In a protocol where the sending & the relay requesting are not separated, what should this function do?

Reply:
In this case, `sendMessage` would not actually interact with the protocol, the message would be stored until `requestRelay` is invoked. Alternatively, the `requestRelay` parameters are included as an attribute and it can all be done in one go.

Reply:
Sounds like the `sendMessage` function would be quite the misnomer then 😄 

Why separate the sendMessage & relay functionality in this standard at all? I know this closely follows the Axelar workflow so maybe @cjcobb23 can comment